src/sentry/utils/pytest/relay.py

```diff
@@ -139,7 +139,23 @@ def relay_server(relay_server_setup):
 
 @pytest.fixture
 def get_relay_store_url(relay_server):
-    def relay_store_url(project_id):
+    def inner(project_id):
         return "{}/api/{}/store/".format(relay_server["url"], project_id)
 
-    return relay_store_url
+    return inner
+
+
+@pytest.fixture
+def get_relay_minidump_url(relay_server):
+    def inner(project_id):
+        return "{}/api/{}/minidump/".format(relay_server["url"], project_id)
```

Comment:
Strictly speaking, this is missing the `?sentry-key=` query parameter, when compared to the URL we show in project settings. That said, this works because we upload the minidump using an `x-sentry-auth` header. One of two options:

1. You leave it like this, but add a comment about this here.
2. (preferred) Add the key here and remove the auth header from `post_and_retrieve_minidump`. We do the same for unreal, so this one seems more consistent.